calico/etcddriver/driver.py

```diff
@@ -78,6 +80,8 @@
 
 # Threshold in seconds for detecting watcher tight looping on exception.
 REQ_TIGHT_LOOP_THRESH = 0.2
+# How often to log stats.
+STATS_LOG_INTERVAL = 10
```

Comment:
I'm slightly nervous about this interval. Log stats every 10s? Maybe in debug mode, but feels to me like every 5 minutes is more like it, maybe every minute to give us better granularity. For now, let's leave this, but feels like we could be buried in log spam if we don't back off a bit.